### DISCUSSION TITLE
cmp window not manually opening without typing (`cmp.mapping.complete()`)

### DISCUSSION THREAD

**Discussion by timii** [hrsh7th/nvim-cmp#1278] [Q&A]
Hello everyone 👋 

I currently have an issue manually showing the nvim-cmp window when not typing. Usually, the nvim-cmp window should show up when pressing `<C-Space>` without having to type anything, but for whatever reason, nothing happens when I press `<C-Space>` currently. 

My nvim-cmp setup:
```lua
cmp.setup({
    snippet = {
        expand = function(args)
            luasnip.lsp_expand(args.body) -- For `luasnip` users.
        end,
    },

    -- Keymappings for the cmp window
    mapping = cmp.mapping.preset.insert({
        ["<C-d>"] = cmp.mapping.scroll_docs(-4),
        ["<C-f>"] = cmp.mapping.scroll_docs(4),
        ["<C-Space>"] = cmp.mapping.complete(),
        ["<CR>"] = cmp.mapping.confirm({
            behavior = cmp.ConfirmBehavior.Replace,
            select = true,
        }),

        ["<C-e>"] = cmp.mapping({
            i = cmp.mapping.abort(),
            c = cmp.mapping.close(),
        }),
        ["<Tab>"] = cmp.mapping(function(fallback)
            if cmp.visible() then
                cmp.select_next_item()
            elseif luasnip.expand_or_jumpable() then
                luasnip.expand_or_jump()
            elseif check_backspace() then
                fallback()
            else
                fallback()
            end
        end, { "i", "s" }),
        ["<S-Tab>"] = cmp.mapping(function(fallback)
            if cmp.visible() then
                cmp.select_prev_item()
            elseif luasnip.jumpable(-1) then
                luasnip.jump(-1)
            else
                fallback()
            end
        end, { "i", "s" }),
    }),

    formatting = {
        -- formatting options
    },
    sources = {
        -- sources
    },
    confirm_opts = {
        behavior = cmp.ConfirmBehavior.Replace,
        select = false,
    },
})

-- `/` cmdline setup.
cmp.setup.cmdline("/", {
    mapping = cmp.mapping.preset.cmdline(),
    sources = {
        { name = "buffer" },
    },
})

-- `:` cmdline setup.
cmp.setup.cmdline(":", {
    mapping = cmp.mapping.preset.cmdline(),
    sources = cmp.config.sources({
        { name = "path" },
    }, {
        { name = "cmdline" },
    }),
})
```
My nvim version:
```bash
NVIM v0.8.0-1210-gd367ed9b2
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Übersetzt von runneradmin@fv-az177-603
Features: -acl +iconv +tui
```

Any help or idea is greatly appreciated and if more information about my config is needed, I will add it in a comment.

Thanks

**Comment by sQVe** [comment]
I'm seeing this issue too. Have you possibly been able to fix it?

**Reply by timii** [reply]
Sadly no. Do you any other idea?

**Reply by sQVe** [reply]
I think the support for it is removed. See https://github.com/hrsh7th/nvim-cmp/discussions/1358.

**Reply by timii** [reply]
That could be the case. If so, that would be really sad if there is no other solution to this problem. Maybe a different solution to this problem has been implemented instead.

**Comment by fitrh** [comment]
Does using another keymap e.g., `<C-l>` also not work?

**Reply by kjgen** [reply]
I have the same issue. `<C-Space>` doesn't work but if I update my config and put `<C-p>` instead of `<C-Space>`, the complete() window shows up. Anyone know why doesn't work? Doing :verbose imap `<C-Space>` shows that nvim-cmp is finding the mapping.

**Reply by kjgen** [reply]
I was able to fix my issue. My issue was associated to OpenSSH version on windows not handling input correctly. I was on 8.1.0.0 of OpenSSH on windows. I upgraded to the beta 9.1.0.0 beta on windows and now the `<C-Space>` is being forwarded correctly through my ssh session

**Reply by carlesba** [reply]
`C-Space` doesn't work for me either. `C-o` instead, works as expected. I can't find them in my list of maps

**Reply by Shougo** [reply]
You can use C-@ mapping instead.
It depends on your terminal.
